### ISSUE TITLE
Zero accuracy 

### ISSUE THREAD

**Issue by firesans** [Holmeyoung/crnn-pytorch#38]
Hi, I have used the same code but for a different language. The issue is I'm getting a zero accuracy for all the epochs and the network is barely learning anything.
Please help me out.
I've attached my training output file.
[slurm-synthetic_train-output-23-01-20_new.txt](https://github.com/Holmeyoung/crnn-pytorch/files/4107738/slurm-synthetic_train-output-23-01-20_new.txt)

**Comment by Holmeyoung** [comment]
Hi, i am so sorry to reply now. Have you fixed it? The loss did't reduce every epoch, maybe you can
Use smaller lr
Use more training data and try again.

**Comment by firesans** [comment]
Hey, yes I fixed that issue of zero accuracy (I've used a larger dataset) but my network is experiencing a lot of fluctuations. The validation loss increases and decreases within each epoch. 
What could be the reason? (I've attached the training output and the plot of validation accuracy per epoch)
[output_file.txt](https://github.com/Holmeyoung/crnn-pytorch/files/5118313/output_file.txt)
![newplot (7)](https://user-images.githubusercontent.com/28893148/91054506-69c4f680-e641-11ea-89c0-df12785d8357.png)
